Sources/ConfidenceProvider/Apply/FlagApplierWithRetries.swift

```diff
@@ -119,7 +122,8 @@ final class FlagApplierWithRetries: FlagApplier {
             flags: applyFlagRequestItems,
             sendTime: Date.backport.nowISOString,
             clientSecret: options.credentials.getSecret(),
-            resolveToken: resolveToken
+            resolveToken: resolveToken,
+            sdk: Sdk(id: metadata.name ?? "SDK_ID_SWIFT_PROVIDER", version: metadata.version ?? "Unknown")
```

Comment:
I'd be tempted to move the default values into the constructor of the `Sdk` itself, something like:
```
init(id: String?, version:String?) {
    self.id = id ?? "SDK_ID_SWIFT_PROVIDER"
    self.version = version ?? "unknown"
}
```